Harvest.Web/ClientApp/package.json

```diff
@@ -16,11 +16,16 @@
     "typescript": "^4.0.5",
     "web-vitals": "^0.2.4"
   },
+  "devDependencies": {
+    "@rescripts/cli": "0.0.15",
+    "cross-env": "^7.0.3",
+    "jest-trx-results-processor": "^2.2.0"
+  },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "rescripts  start",
```

Comment:
Is rescripts just informational? I'd say react-scripts is more descriptive

Reply:
yea i agree I don't like that we had to add a whole rescripts cli package just to get code coverage to work.  Reading the repo for rescripts it sounds like something I'd like to avoid having

Reply:
I see why you had to do it since CRA doesn't support the results processor but hopefully there is another option

Reply:
Can you just add the reporter in the CLI `npm run citest` config directly, like `--reports jest-trx-results-processor` or something?

Reply:
I had done it that way because there doesn't appear to be a way to override default configs for `jest-trx-results-processor` via cli. I wanted both trx and coverage files to be located in the output folder. They're now being placed in the ClientApp folder.